### PR TITLE
Implement stickers

### PR DESCRIPTION
**Pull request by asleep-cult**
Adds `StickerState` (`Client.stickers`), `GuildStickerState` (`Guild.stickers`), `StickerPack` (`StickerState.fetch_packs`), `StickerItem` (`Message.sticker_items`), `StandardSticker` and `GuildSticker`

### CODE REVIEW

File: snekcord/clients/wsevents.py [hunk 1]
```diff
@@ -201,7 +201,31 @@ async def execute(cls, client, shard, payload):
...
+    async def execute(cls, client, shard, payload):
+        guild = client.guilds.get(Snowflake(payload['guild_id']))
+
+        if guild is not None:
+            stickers = set()
+
+            for sticker in payload['stickers']:
+                stickers.add(guild.stickers.upsert(sticker).id)
+
+            for sticker_id in set(guild.stickers.keys()) - stickers:
```
Comment (ToxicKidz) [inline snekcord/clients/wsevents.py:227]:
```suggestion
            for sticker_id in guild.stickers.keys() - stickers:
```

File: snekcord/states/messagestate.py [hunk 2]
```diff
@@ -69,11 +69,29 @@ async def create(
             json['tts'] = bool(tts)
 
         if embed is not None:
+            if 'embeds' not in json:
+                json['embeds'] = []
```
Comment (ToxicKidz) [inline snekcord/states/messagestate.py]: Is this worth checking? At this point this condition will never be false

**Review by ToxicKidz — CHANGES_REQUESTED** [review]
Are you going to make stubs for this?

File: snekcord/states/messagestate.py (hunk 2 above)
Comment (asleep-cult) [reply snekcord/states/messagestate.py]: `json['embeds']` will only exist if `embed` is provided :thinking:

Comment (ToxicKidz) [reply snekcord/states/messagestate.py]: but in the `embed` if statement, it doesn't have that key

File: snekcord/clients/wsevents.py (hunk 1 above)
Comment (asleep-cult) [reply snekcord/clients/wsevents.py:227]: I don't want to change this yet, I might change `keys()` to return something else in the future

Comment (ToxicKidz) [reply snekcord/clients/wsevents.py:227]: oh ok

**Comment by asleep-cult** [comment]
> Are you going to make stubs for this?

Eventually :neckbeard:

**Review by ToxicKidz — APPROVED** [review]

**Pull request merged**